contracts/src/v0.8/functions/v1_1_0/libraries/ChainSpecificUtil.sol

```diff
@@ -46,12 +45,14 @@ library ChainSpecificUtil {
   /// @notice On Arbitrum, the provided calldata is not used to calculate the fees.
   /// @notice On Optimism, the provided calldata is passed to the GasPriceOracle predeploy
   /// @notice and getL1Fee is called to get the fees.
-  function _getCurrentTxL1GasFees(bytes memory txCallData) internal view returns (uint256 l1FeeWei) {
+  function _getL1DataGasCostUpperLimit(bytes memory dataSizeBytes) internal view returns (uint256 l1FeeWei) {
```

Comment:
`dataSizeBytes` implies it's a size number (which I agree is a better way), but the type is still bytes here.